packages/webdriverio/tests/commands/element/shadow$$.test.js

```diff
@@ -1,5 +1,13 @@
 import { remote } from '../../../src'
 
+jest.mock('../../../src/commands/element/$$', () => ({
+    __esModule: true,
+    default: jest.fn().mockImplementation(() => { })
+}))
+
+const $$ = require('../../../src/commands/element/$$')
+
+
```

Comment:
should we have here only one new line instead of two?

Reply:
Fixed.